nvtabular/framework_utils/tensorflow/layers/__init__.py

```diff
@@ -0,0 +1,4 @@
+from embedding import ScalarDenseFeatures, ScalarLinearFeatures
```

Comment:
Can you add a copyright notice to all the python files ?
```suggestion
#
# Copyright (c) 2020, NVIDIA CORPORATION.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
from embedding import ScalarDenseFeatures, ScalarLinearFeatures
```

Reply:
They should all have them as of c4dceffd1664c23ac0aed9426735dc200c096649, but let me know if I'm missing any